overview/default.nix

```diff
@@ -301,25 +301,51 @@ let
         ${heading 2 "demo" "Run a demo deployment locally"}
 
         <ol>
-          <li><strong>Install Nix on your platform.</strong></li>
           <li>
-            <strong>Download this Nix file to your computer.</strong>
-            It obtains the NGIpkgs source code and declares a basic service configuration
-            to be run in a virtual machine.
-            ${render.codeSnippet.one {
-              filename = "default.nix";
-              relative = true;
-              downloadable = true;
-            }}
+            <details>
+            <summary><strong>Install Nix</strong></summary>
+              <ul>
+                <li>Arch Linux</li>
+                  <pre><code>pacman --sync --refresh --noconfirm curl git jq nix</code></pre>
+                <li>Debian/Ubuntu</li>
+                  <pre><code>apt install --yes curl git jq nix</code></pre>
+              </ul>
+            </details>
           </li>
           <li>
-            <strong>Build the virtual machine</strong> defined in <code>default.nix</code> and run it:
-            <pre><code>nix-build && ./result</code></pre>
-            Building <strong>will</strong> take a while.
+            <details>
+            <summary><strong>Download a configuration file</strong></summary>
+              ${render.codeSnippet.one {
+                filename = "default.nix";
+                relative = true;
+                downloadable = true;
+              }}
+            </details>
           </li>
           <li>
-            <strong>Access the service</strong> with a web browser:
-            <a href="http://localhost:${toString servicePort}">http://localhost:${toString servicePort}</a>
+            <details>
+            <summary><strong>Enable binary substituters</strong></summary>
```

Comment:
We could make this optional (and hidden in a `detail`), but then we should provide motivation why to do it:
```suggestion
            <summary><strong>Enable binary substituters (optional, for less wait times)</strong></summary>
```

Reply:
> Run a demo deployment locally could be a detail

Yes, that's better. I agree.

Reply:
> Run a demo deployment locally could be a detail

Done in https://github.com/ngi-nix/ngipkgs/pull/1013/commits/57aa464d1b04526d0d364f266c691508cd2c8788

Reply:
* collapsed
![screen-imincik-2025-05-15-08:18:49](https://github.com/user-attachments/assets/160bef3a-2d1b-494b-a0d4-681252799397)

* not collapsed
![screen-imincik-2025-05-15-08:18:58](https://github.com/user-attachments/assets/6311cd05-3705-4c24-8d3e-bf359d9de26a)